src/EPPlus/FormulaParsing/Excel/Functions/Finance/Implementations/OddfpriceImpl.cs

```diff
@@ -0,0 +1,225 @@
+﻿/*************************************************************************************************
+  Required Notice: Copyright (C) EPPlus Software AB. 
+  This software is licensed under PolyForm Noncommercial License 1.0.0 
+  and may only be used for noncommercial purposes 
+  https://polyformproject.org/licenses/noncommercial/1.0.0/
+
+  A commercial license to use this software can be purchased at https://epplussoftware.com
+ *************************************************************************************************
+  Date               Author                       Change
+ *************************************************************************************************
+  26/06/2023         EPPlus Software AB           EPPlus v7
+ *************************************************************************************************/
+
+using OfficeOpenXml.FormulaParsing.Excel.Functions.Finance.FinancialDayCount;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OfficeOpenXml.FormulaParsing.Excel.Functions.Finance.Implementations
+{
+    internal class OddfpriceImpl
+    {
+        private readonly System.DateTime _settlementDate;
```

Comment:
Add a `using System;` in the top of the file  and remove the namespace references on `DateTime`

Reply:
In Epplus formulaparser there already is a namespace called DateTime (fficeOpenXml.FormulaParsing.Excel.Functions.DateTime), it collides with System.DateTime. To resolve we would need to change that namespace, but that is maybe not in the scope of this pull request?

Reply:
Mats will rename this namespace in another commit.